### PR TITLE
Remove .NET Core 3.1 and .NET 5 TFM

### LINKED ISSUE #3972: Remove .NET Core 3.1 TFM
.NET Core 3.1 out of support  December 13, 2022

### PR DESCRIPTION
**Pull request by Marusyk**
Fixes #3972 and #3949

Please review
Thank you in advance

### CODE REVIEW

File: appveyor.yml [hunk 1]
```diff
@@ -7,8 +7,6 @@ init:
 build_script:
   - ps: Invoke-RestMethod -Uri 'https://dot.net/v1/dotnet-install.ps1' -OutFile '.\dotnet-install.ps1'
```
Comment (Marusyk) [inline appveyor.yml:8]: Do we need `dotnet-install.ps1` after the changes?

Comment (devlead) [reply appveyor.yml:8]: Yes, as .NET 7 is in global.json we should install .NET 6.0, so remove 3.1 and replace 5.0 with 6.0.

**Comment by Marusyk** [comment]
thanks @devlead
sorry, was busy

**Review by devlead — APPROVED** [review]
LGTM 👍

**Comment by devlead** [comment]
@Marusyk your changes have been merged, thanks for your contribution 👍

**Pull request merged**